### DISCUSSION TITLE
How and where can I host an online experiment?

### DISCUSSION THREAD

**Discussion by Ricardo-Piccolo** [jspsych/jsPsych#3052] [Q&A]
Hello,

I have to build some experiments for my scholarship and then host them somewhere for people to do it online for free. I've already done the building part, but now I cannot host the experiments I created on JsPsych. Can you guys help me, please?

**Comment by jodeleeuw** [comment]
One option for a totally free solution is [datapipe](https://pipe.jspsych.org/getting-started). This will involve hosting your experiment on GitHub Pages and storing the data on the Open Science Framework.

**Reply by Ricardo-Piccolo** [reply]
I just tried that, following every single step the website proposes, but it just doesn't work. :(
The link should be: https://Ricardo-Piccolo.github.io/GoNoGo **OR** https://Ricardo-Piccolo.github.io/GoNoGo/go_no_go.html
But it keeps telling that it does not exist
:(

**Reply by jodeleeuw** [reply]
Did you turn on GitHub Pages for the repo, following this part of the instructions?

> Next, configure the repository to share its content as a webpage. Go to the Settings tab of your repository. Select the Pages menu item on the left side. For Source, leave it as deploy from a branch. Under branch select main as the source. Click the save button to finish this step.
